Aquality.Selenium/src/Aquality.Selenium/Browsers/IBrowserTabNavigation.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+
+namespace Aquality.Selenium.Browsers
+{
+    /// <summary>
+    /// Provides functionality to work with browser tab navigation.  
+    /// </summary>
+    public interface IBrowserTabNavigation
+    {
+        /// <summary>
+        /// Gets current tab handle.
+        /// </summary>
+        /// <returns>Current tab handle.</returns>
+        string CurrentTabHandle { get; }
+
+        /// <summary>
+        /// Gets opened tab handles.
+        /// </summary>
+        /// <returns>List of tab handles.</returns>
+        IList<string> TabHandles { get; }
+
+        /// <summary>
+        /// Switches to tab.
+        /// </summary>
+        /// <param name="name">Tab handle.</param>
+        /// <param name="closeCurrent">Close current tab if true and leave it otherwise.</param>
+        void SwitchToTab(string tabHandle, bool closeCurrent = false);
+
+        /// <summary>
+        /// Switches to tab.
+        /// </summary>
+        /// <param name="index">Tab index.</param>
+        /// <param name="closeCurrent">Close current tab if true and leave it otherwise.</param>
+        void SwitchToTab(int index, bool closeCurrent = false);
+
+        /// <summary>
+        /// Switches to tab.
```

Comment:
Maybe add some details to summary such as: switches to the last tab

Reply:
Do you think we need to correct method name like SwitchesToLastTab accordingly

Reply:
Probaly. But SwitchToLastTab

Reply:
done